tests/functionaltests/Tests/NSURLSession.mm

```diff
@@ -673,14 +670,12 @@ inline int _GetLastDelegateCall(NSURLSessionDownloadTaskTestHelper* downloadTask
     [conditionCancelled unlock];
 
     // Make sure download has stopped.
-    lastBytesDownloaded = (double)downloadTaskTestHelper.totalBytesWritten / 1024 / 1024;
     [NSThread sleepForTimeInterval:0.5];
```

Comment:
PLEASE PLEASE PLEASE NO.

This is *NOT* a good way to test this sort of thing. These tests should honestly be scrapped and rewritten using events / mocks / something so we aren't guessing on when it will complete. It just slows down things for everyone and only adds minimal more reliability because they still hit real endpoints that could take arbitrary time to respond ....